user/serializers.py

```diff
@@ -1 +1,27 @@
-# write your code here
+from django.contrib.auth import get_user_model
+from rest_framework import serializers
+
+
+class UserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = get_user_model()
+        fields = ("id", "username", "email", "password", "is_staff")
+        read_only_fields = ("id", "is_staff")
+        extra_kwargs = {
+            "password": {"write_only": True,
+                         "min_length": 5
+                         }
+        }
+
+    def create(self, validated_data):
+        return get_user_model().objects.create_user(**validated_data)
```

Comment:
Ensure that the `create_user` method correctly handles password hashing. The `create_user` method of the user model should automatically handle this, but it's important to verify that this is the case.